.github/workflows/regression-report.yml

```diff
@@ -222,10 +225,58 @@ jobs:
         run: |
           ./.ci/append-to-github-output.sh "message" "$(cat .ci-temp/message)"
 
+  check_cancellation:
+    needs: [parse_comment, make_report]
+    if: always()
+    runs-on: ubuntu-latest
+    outputs:
+      cancelled: ${{ steps.check_status.outputs.cancelled }}
+    steps:
+      - name: Check job status
+        id: check_status
+        run: |
+          if [ "${{ contains(needs.*.result, 'cancelled') }}" = "true" ]; then
+            echo "Job was cancelled due to concurrency"
+            echo "cancelled=true" >> $GITHUB_OUTPUT
+          else
+            echo "Job was not cancelled"
+            echo "cancelled=false" >> $GITHUB_OUTPUT
+          fi
+
+      - name: Change reaction if cancelled
+        if: steps.check_status.outputs.cancelled == 'true'
+        uses: actions/github-script@v7
+        with:
+          github-token: ${{secrets.GITHUB_TOKEN}}
+          script: |
+            // Remove the rocket reaction
+            const reactions = await github.rest.reactions.listForIssueComment({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              comment_id: context.payload.comment.id,
+            });
+            const rocketReaction = reactions.data.find(reaction => reaction.content === 'rocket');
+            if (rocketReaction) {
+              await github.rest.reactions.deleteForIssueComment({
+                owner: context.repo.owner,
+                repo: context.repo.repo,
+                comment_id: context.payload.comment.id,
+                reaction_id: rocketReaction.id,
+              });
+            }
+            
+            // Add the confused (sad) reaction
+            await github.rest.reactions.createForIssueComment({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              comment_id: context.payload.comment.id,
+              content: 'confused'
+            });
+
   # should be always last step
   send_message:
     runs-on: ubuntu-latest
-    needs: [ parse_comment, make_report ]
+    needs: [ check_pr_status, parse_comment, make_report ]
```

Comment:
Let's not send messages on cancellation. No chat required for this line

Reply:
done, just addding an emoji here if cancelled